src/Backend/OnboardingInsuranceAPI/Enums/CarTypesContribution.cs

```diff
@@ -0,0 +1,37 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace OnboardingInsuranceAPI.Enums
+{
+    public enum CarTypes
+    {
+        Cabriolet = 500,
+        SportCar = 1000,
+        Van = 200,
+        Micro = 50,
+        Sedan = 150,
+        CamperVan = 500,
+        SuperCar = 2000,
+        Coupe = 200,
+        Other = 100
```

Comment:
These values are weird. You should always have them sorted from the "default" and then by the number.